app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/AddressController.scala

```diff
@@ -172,7 +171,7 @@ class AddressController @Inject()(
 
   private def storeAddress(address: Address)(implicit request: RequestWithSubscriptionReady[_]): Future[Result] =
     updateSession(sessionStore, request)(
-      _.copy(subscriptionStatus = Some(addressLens.set(request.subscriptionReady)(address)))
+      _.copy(journeyStatus = Some(addressLens.set(request.subscriptionReady)(address)))
     ).map(
```

Comment:
This is confusing - on the one hand an address is being set but it is being assigned to an identifier which has name subscriptionStatus or now journeyStatus - can this be improved to make the intention clear?

Reply:
the subscriptionReady is being updated with a new address with the `Lens` called `addressLens` - if it's too confusing I can take the lens stuff out to make it clearer what's going on. I just included it to avoid having to do stuff like `A.copy(a = A.a.copy(...))`

Reply:
do you reckon this is too ugly?
```scala
      _.copy(journeyStatus = Some(request.subscriptionReady.copy(
        subscriptionDetails = request.subscriptionReady.subscriptionDetails.copy(
          address = address
        ))))
```

Reply:
no, let's keep it in. Lens are fine. I looked at monocle and I don't think we will get the syntax any terser than what you have here already.